Building broken - cannot make

Cannot build it. First try results in:
```
gofmt -w `find ./ -name '*.go'`
mkdir -p bin
go build -o bin/resource-constants utils/resource-constants.go
if [ ! -e "resources" ]; then mkdir resources; fi
bin/resource-constants --base_dir /my/local/path/to/todo-tracks/ui/ > resources/constants.go
go test ./...
pattern ./...: directory prefix . does not contain main module or its selected dependencies
make: *** [test] Error 1
```

Make only succeeds if I first run this command:
```
go mod init "github.com/google/todo-tracks"
```

--- a/go.mod
+++ b/go.mod
@@ -1,0 +1,3 @@
+module github.com/google/todo-tracks
+
+go 1.14

